recipes/gfaviz/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "gfaviz" %}
+{% set version="1.0.0" %}
+{% set hash="4bc4327dd182812a9cfd53a39632bf41f461170e142df62ebbd0ba441239a6d4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True # [ windows ]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x") }}
+  script:
+    - qmake
+    - make
+    - mkdir -p {{ PREFIX }}/bin
+    - cp {{ SRC_DIR }}/gfaviz.app/Contents/MacOS/gfaviz {{ PREFIX }}/bin/ # [ osx ]
+    - cp {{ SRC_DIR }}/gfaviz {{ PREFIX }}/bin/ # [ linux ]
+
+source:
+  url: https://github.com/ggonnella/gfaviz/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - qt =5
+    - libgl-devel # [ linux ]
```

Comment:
Libraries should be in 'host' dependencies

Reply:
I moved `qt` and `libgl` libs to `host` part